### ISSUE TITLE
[Question] Iframe Jquery API configuration

### ISSUE THREAD

**Issue by AmKang** [ColorlibHQ/AdminLTE#3536]
Hi,I tried to use the new version of Iframe function, but when I wanted to use jQuery configuration, I encountered some problems.
This configuration has no effect. When you click on the menu and a new tab is created, it does not output content.

/********************************
$('.content-wrapper').IFrame({
  onTabClick(item) {
    console.log("onTabClick")
  },
  onTabChanged(item) {
    console.log("onTabChanged")
  },
  onTabCreated(item) {
    console.log("onTabCreated")
  },
  autoIframeMode: true,
  autoItemActive: true,
  autoShowNewTab: true,
  allowDuplicates: true,
  loadingScreen: 750,
  useNavbarItems: true
})
******************************/

**Environment (please complete the following information):**
- AdminLTE Version: 1 v3.1.01
- Operating System:   Window 
- Browser (Version):  Chrome 

Thank you,Best wishes.

**Comment by danny007in** [comment]
use under 
```js
// Shorthand for $( document ).ready()
$(function() {
    console.log( "ready!" );
});
```

**Comment by danny007in** [comment]
rename issue title

**Comment by AmKang** [comment]
I mean, I configure the Iframe through the jQuery API. I hope that when a new Iframe TAB  is created, something can be output, but according to the configuration, it does not output anything in the console.
I hope that when the Iframe TAB is created, it can output the content, console.log("onTabCreated")

`
/********************************
$('.content-wrapper').IFrame({  
onTabClick(item) {
console.log("onTabClick")
},
onTabChanged(item) {
console.log("onTabChanged")
},
onTabCreated(item) {
console.log("onTabCreated")
},
autoIframeMode: true,
autoItemActive: true,
autoShowNewTab: true,
allowDuplicates: true,
loadingScreen: 750,
useNavbarItems: true
})
******************************/
`

Best wishes.

**Comment by AmKang** [comment]
I don't need loading. I use the following configuration, but it doesn't work.

`$(function() {
    $('.content-wrapper').IFrame(".content-wrapper",{
	  loadingScreen: false //
	})
});`
![QQ截图20210331165516](https://user-images.githubusercontent.com/43641341/113118816-68485600-9242-11eb-841d-238a99569528.png)

**Comment by AmKang** [comment]
I don't need loading. I use the following configuration, but it doesn't work.

`$(function() {
    $('.content-wrapper').IFrame({
	  loadingScreen: false
	})
});`

![QQ截图20210331165516](https://user-images.githubusercontent.com/43641341/113119163-b3626900-9242-11eb-84c5-a50a8b823b98.png)

**Comment by REJack** [comment]
Fixed with 5b07bdd. 😄
